uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/InstallBrokerTestRule.java

```diff
@@ -0,0 +1,56 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation.rules;
+
+import androidx.annotation.Nullable;
+
+import com.microsoft.identity.client.ui.automation.broker.ITestBroker;
+
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+/**
+ * A Test Rule to install the provided broker on the device prior to executing the test case
+ */
+public class InstallBrokerTestRule implements TestRule {
+
+    private final ITestBroker broker;
+
+    public InstallBrokerTestRule(@Nullable ITestBroker broker) {
```

Comment:
I was trying to use as shortcut for both broker and non-broker tests. So non-broker tests can just say broker is null and this would install anything. But you're right, this should be `NonNull`. I'll clean this up.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/982/commits/b210f1b6280abec3158cf68a5e5b5b583819a46d